src/protocols/kubernetes/argv.c

```diff
@@ -21,6 +21,7 @@
 #include "argv.h"
 #include "kubernetes.h"
 #include "terminal/terminal.h"
+#include "terminal/terminal_priv.h"
```

Comment:
Why is the private internal header included here? I'd think it'd be included and required only by the terminal internals.

Reply:
Because internals from the terminal struct are used here - see line 57. The same goes for all the others that have the header included.

Reply:
I guess it's not very internal if it's used all over the place like this, but there sure is a lot of code that's reaching into the terminal internals right now...

Reply:
Ok, should be better now - nothing from `terminal_priv.h` is used outside of the terminal code itself.